crates/matrix-sdk-crypto/src/verification/cache.rs

```diff
@@ -105,11 +105,21 @@ impl VerificationCache {
 
     #[cfg(feature = "qrcode")]
     pub fn insert_qr(&self, qr: QrVerification) {
+        tracing::debug!(
```

Comment:
nit: don't use a fully qualified path but instead `use tracing::debug` at the top.

Reply:
ff484509331180c2be6114b32404485e948db2cf